src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -167,6 +166,22 @@ def setUp(self):
         self.log.info("*** SETUP running on %s ***", str(detect()))
         super().setUp()
 
+        # Random generator that could be seeded for reproducibility
+        env_seed = os.environ.get("DAOS_TEST_RANDOM_SEED", None)
+        if env_seed is None:
+            self.rand_seed = int.from_bytes(os.urandom(8), byteorder='little')
```

Comment:
Thanks for this! FYI, we'll probably also improve in the future to read from a config if needed